plugin/src/utils/gatsby-types.ts

```diff
@@ -1,2 +1,7 @@
+// these aren't typed in Gatsby..
+
+import type { NodePluginArgs, Reporter } from "gatsby"
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export type GatsbyHelpers = any
+export type GatsbyNodeApiHelpers = NodePluginArgs
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Can we remove the comments now? :)

Reply:
yes! why are these comments only appearing in files changed view but not conversation view? so confused? github is gettin' real buggy for me, lol